src/clients/http_client/http_client.ts

```diff
@@ -116,8 +119,10 @@ class HttpClient {
             break;
         }
         headers = {
+          /* eslint-disable @typescript-eslint/naming-convention */
           'Content-Type': type,
-          'Content-Length': Buffer.byteLength(body as string).toString(),
+          'Content-Length': (body as string).length.toString(),
```

Comment:
`Buffer.byteLength` is from Node and it's not available in Cloudflare Worker (another issue caught by the tests!)

Since all the cases in the `switch` preceding this ensure that `body` is set to a `string`, I think it's fairly safe to use the string`.length` method to set the `Content-Length` ... thoughts? 🤔

Reply:
No, not quite, because JavaScript strings are UCS-2 encoded, but strings will be UTF-8 encoded when sent over the wire by most (all?) web server implementations. Those can have different lengths for the same string. For example:

```js 
"👪".length // => 2
new TextEncoder("utf8").encode("👪").length // => 4
```

I’d recommend we use the latter to set `Content-Length`.